src/locales/ja.ts

```diff
@@ -1,3 +1,3 @@
 export default {
-  back: 'Retour',
+  back: 'jBack',
```

Comment:
is this change intentional?

Reply:
I mean, sort of 🤷 

I renamed the fr file to ja and just threw that in there for the moment. Should I copy over the English strings for now?

Reply:
oh i see. i guess i would delete anything that's not currently being used and let it fall back to the english translation

Reply:
I removed the key for now